exercises/phone-directory-cpp/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.10)
 
 set (CMAKE_BUILD_TYPE Debug)
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-set (CMAKE_CXX_COMPILER "D:\\mingw\\bin\\mingw32-g++.exe")
```

Comment:
It's better to have this set in runtime, path is relative to the user

Reply:
Fixed